src/vscode-cosmos.api.d.ts

```diff
@@ -0,0 +1,30 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export interface CosmosDBItem {
+    accountName: string | undefined
+    connectionString: string | undefined
```

Comment:
None of these should be `| undefined`. getDatabase and pickDatabase should always return an item with these properties defined.